index.html

```diff
@@ -122,288 +122,79 @@ <h2 class="item2">Multi-Post Stories Gain+Glory</h2>
-    <div class="main-grid">
-      <div class="flex1">
+            <a href="#" target="_blank" class="item7">Get my resume</a>
+     </div>
+     <div class="main-grid">
+       <div class="flex1">
           <img id="icon" src="./features/images/Icons.png">
-         <p class="sub-head">Languages</p>
-        <ul class="flex2">
-        <li class="lang">JavaScript</li>
-        <li class="lang">Ruby</li>
-        <li class="lang">HTML</li>
-        <li class="lang">CSS</li>
-        </ul>
-      </div>
-      <div class="flex1">
-       <img id="icon" src="./features/images/Icon.png">
-      <p class="sub-head">Frameworks</p>
-     <ul class="flex3">
-     <li class="lang">React.js</li>
-     <li class="lang">Ruby on Rails</li>
-     <li class="lang">RSpec</li>
-     </ul>
-     <ul class="flex4">
-       <li class="lang">Capybara</li>
-       <li class="lang">Selenium</li>
-     </ul>
-   </div>
-   <div class="flex1">
-       <img id="icon" src="./features/images/ico.png">
-      <p class="sub-head">Skills</p>
-     <ul class="flex3">
-     <li class="lang">Database Management</li>
-     <li class="lang">Version Control</li>
-     </ul>
-     <ul class="flex4">
-       <li class="lang">CLI</li>
-       <li class="lang">Web development</li>
-       <li class="lang">API Design</li>
-     </ul>
-   </div>
-    </div>
-    <div class="flex5">
+            <p class="sub-head">Languages</p>
+             <ul class="flex2">
+              <li class="lang">JavaScript</li>
+              <li class="lang">Ruby</li>
+              <li class="lang">HTML</li>
+              <li class="lang">CSS</li>
+             </ul>
+       </div>
+       <div class="flex1">
+         <img id="icon" src="./features/images/Icon.png">
+          <p class="sub-head">Frameworks</p>
+            <ul class="flex3">
+             <li class="lang">React.js</li>
+             <li class="lang">Ruby on Rails</li>
+             <li class="lang">RSpec</li>
+           </ul>
+            <ul class="flex4">
+              <li class="lang">Capybara</li>
+              <li class="lang">Selenium</li>
+           </ul>
+        </div>
+       <div class="flex1">
+         <img id="icon" src="./features/images/ico.png">
+          <p class="sub-head">Skills</p>
+           <ul class="flex3">
+            <li class="lang">Database Management</li>
+            <li class="lang">Version Control</li>
+           </ul>
+          <ul class="flex4">
+           <li class="lang">CLI</li>
+           <li class="lang">Web development</li>
+           <li class="lang">API Design</li>
+         </ul>
+       </div>
+     </div>
+      <div class="flex5">
         <p id="bio">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
        </div>
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
```

Comment:
- [ ] `README.md` file: To make your README be more professional and descriptive, kindly make sure you fix the broken link at the top of your README. I suggest you completely remove the broken logo. But Optionally, the you can replace with your own logo if you have one.
![Screenshot from 2023-06-07 11-29-30](https://github.com/Bupilipili/MY-PORTFOLIO/assets/83042742/acd01e36-a539-4bf7-85d4-d21a4d663aed)